lib/vdsm/storage/lvm.py

```diff
@@ -657,10 +660,20 @@ def _reload_single_vg(self, vg_name):
             if error:
                 self._update_stale_vgs([vg_name])
 
+                if not out:
```

Comment:
I'm not sure it is possible to get an error from vgs and some output, so we can simplify by raising on errors.